ext/ddtrace.stub.php

```diff
@@ -0,0 +1,496 @@
+<?php
+
+/** @generate-class-entries */
+
+// phpcs:disable PSR1.Classes.ClassDeclaration.MultipleClasses
+
+namespace DDTrace {
+
+    class SpanData {
+        /**
+         * @var string The span name (Optional as of ddtrace v0.47.0; defaults to ‘ClassName.methodName’ if not set)
```

Comment:
Let's only talk about the current version :-)
Thta's only the case for spans created via install_hook/trace_method/trace_function. If you do start_span(), then it's empty.